packages/ui/src/components/va-option-list/VaOptionList.vue

```diff
@@ -150,9 +131,11 @@ export default defineComponent({
 
     const { computedError, computedErrorMessages } = useValidation(props, emit, reset, focus)
 
+    const computedProps = computed(() => pick(props, ['name', 'color', 'readonly', 'leftLabel']))
+
     onMounted(() => {
-      if (!valueComputed.value && props.defaultValue) {
-        selectedValue.value = props.defaultValue
+      if (props.type !== 'radio' && !Array.isArray(props.modelValue) && process.env.NODE_ENV !== 'production') {
```

Comment:
```suggestion
      if (process.env.NODE_ENV !== 'production' && props.type !== 'radio' && !Array.isArray(props.modelValue)') {
```

even better create a function isDev somewhere in utils

```suggestion
      if (isDev && props.type !== 'radio' && !Array.isArray(props.modelValue)) {
```